skyreader/result.py

```diff
@@ -203,7 +204,7 @@ def isclose_pixel(
 
     def has_metadata(self) -> bool:
         """Check that the minimum metadata is set."""
-        for mk in "run_id event_id mjd event_type nside".split():
+        for mk in self.METADATA_KEYS:
```

Comment:
Could you call this `has_minimum_metadata` and `MINIMUM_METADATA_KEYS`?

Reply:
Yes, although I'll go with "minimal" rather than "minimum".